### ISSUE TITLE
Question for switching multiple cameras

### ISSUE THREAD

**Issue by coldorbit** [zappar-xr/zappar-react-three-fiber-face-tracking-webpack-bootstrap#52]
HI, Thank you for your great work with Zappar React project.

I'm implementing Zappar React project to Twilio video project and have a question regarding <ZapparCamera> component.

How can I change cameras if users have several cameras : actually not face or rear.

Thank you.

**Comment by nyan-left** [comment]
Hi @miracle-arrow.

Switching between cameras other than rear and front is not currently exposed in Zappar for ThreeJS (or React).
You're able to do this using our lower level js package -  [@zappar/zappar](https://www.npmjs.com/package/@zappar/zappar). 

I understand that this can be an important feature and will be investigating on Monday if this is something that we can support for ThreeJS and React Three Fiber 👍 

Will keep this issue thread updated :)

**Comment by nyan-left** [comment]
Hi again @miracle-arrow,

I have discussed this with the team and we're happy to go forward with this feature request, would expect this to be released in the next update of @zappar/zappar-threejs and @zappar/zappar-react-three-fiber.

Will be closing this issue when the feature's released 👍

**Comment by coldorbit** [comment]
Thank you, @nyan-left , Really appreciated.

**Comment by coldorbit** [comment]
HI, @nyan-left , How are you doing?

I've seen that there are several publishes on the @zappar/zappar-react-three-fiber.

Does this have multi camera switch feature?

Thank you :)

**Comment by nyan-left** [comment]
Hi again @miracle-arrow ,

The recent updates have been some minor bug fixes. I will be updating/closing this issue once we have released the feature :)

**Comment by nyan-left** [comment]
Hi @miracle-arrow ,

We've just released 0.1.27 which will cover this feature for you. 

The camera component now takes a sources prop, in which you can define exact video device IDs you'd like to use.
Here's an example:
```tsx
<ZapparCamera
  sources={{
    userCamera:
      "ff237f94e69bf794a40f862cd1b431e7b8e2231fc11b624b94d40d6ab32c6b16",
    rearCamera:
      "ff237f94e69bf794a40f862cd1b431e7b8e2231fc11b624b94d40d6ab32c6b16",
  }}
/>
```
Hope this helps :)
